bpf/lib/common.h

```diff
@@ -609,9 +609,11 @@ enum {
 	CB_POLICY,
 #define	CB_ADDR_V6_2		CB_POLICY	/* Alias, non-overlapping */
 #define	CB_BACKEND_ID		CB_POLICY	/* Alias, non-overlapping */
+#define CB_SRC_PORT		CB_POLICY	/* Alias, non-overlapping */
```

Comment:
Watch out for the mix of tab and space indentations here.

Reply:
will do.